sigs.yaml

```diff
@@ -2205,6 +2205,53 @@ sigs:
     owners:
     - https://raw.githubusercontent.com/kubernetes-sigs/dashboard-metrics-scraper/master/OWNERS
     - https://raw.githubusercontent.com/kubernetes/dashboard/master/OWNERS
+- dir: sig-usability
+  name: Usability
+  mission_statement: >
+    The scope of SIG usability is the core end-user usability of the Kubernetes project.
+    Thiis covers topics like user experience and accessibility. The goals of the SIG
+    are to ensure that the Kubernetes project is maximaly usable by users of a variety
+    of different backgrounds and abilities. Examples of efforts include internationalization
+    and accessibility of documentation.
+
```

Comment:
I like this mission statement but feel like making something _usable_ does not mean making it a *positive experience* that encourages trust and continued use. 

Suggestion: "The goal of SIG "Usability" is to facilitate adoption of the Kubernetes project by as diverse a community of end users as possible. We do this be ensuring that each end user’s interaction with Kubernetes, from discovery to successful production use is seamless and positive. Examples of efforts include user research, internationalization and accessibility."

Still throwing out SIG End User Experience as a name suggestion since that makes the connection to our CNCF End User community more explicit. Echoing other people's assumptions that this SIG would be responsible for collaborating with other SIGs to give the end users channels for providing feedback on their experience.